portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -124,7 +125,7 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     private String getParameter(HttpServletRequest request, String name, String defaultValue) {
         String value = request.getParameter(name);
         if (value == null) {
-            System.out.println("Could not convert number comments wanted input to int: " + value);
+            System.out.println("No input received for wanted number of site comments.");
```

Comment:
It looks like this getParameter method is used for both the comment and the number of comments.  Probably you could update this message to reflect the usage of either.  Something like:

System.out.println("No input received for " + name);

Reply:
Thanks for the good catch